README.md

```diff
@@ -172,6 +172,9 @@ Available options:
 
 + `-R`: recursive
 + `-A`: all files (include files beginning with `.`, except for `.` and `..`)
++ `-d`: list directories themselves, not their contents
++ `-l`: return an array of objects instead of strings, each with properties
+        resembling `ls -l` output
```

Comment:
@gvn what do you think about this? I think this level of detail is good (we don't need to specify each field). I think the wording could be improved, however.